dist/wasm/browser.js

```diff
@@ -0,0 +1,42 @@
+// This module expects that all custom JS files have already been brought into
```

Comment:
do we want this file under version control? if not, there should be a Makefile rule to delete it

Reply:
same question about the next few files, e.G. main.js, browser.html

Reply:
Good catch ! I only need `loader.js` and `shell.js` to make things work

Reply:
However I left all the files produced by Kremlin (`main.js`, `main.html`, etc) under version control, do you also want me to delete them from git ?

Reply:
Ok, deleting them with a Makefile rule